src/Avalonia.Dialogs/ManagedFileChooser.xaml

```diff
@@ -76,7 +76,7 @@
 
         <ListBox Margin="0 0 5 5" BorderBrush="Transparent" x:Name="QuickLinks" Items="{Binding QuickLinks}"
                  SelectedIndex="{Binding QuickLinksSelectedIndex}"
-                 DockPanel.Dock="Left" Background="{DynamicResource ThemeControlMidBrush}" Focusable="False">
```

Comment:
ListBoxItem already has colored background in Fluent theme.
In Default theme it will have transparent background, still looks good tho.